xarray/testing/assertions.py

```diff
@@ -36,7 +36,7 @@ def wrapper(*args, **kwargs):
 
 def _decode_string_data(data):
     if data.dtype.kind == "S":
-        return np.core.defchararray.decode(data, "utf-8", "replace")
+        return np.char.chararray.decode(data, "utf-8", "replace")
```

Comment:
I think we should use the function instead of the method:
```suggestion
        return np.char.decode(data, "utf-8", "replace")
```

Reply:
Good point, done!